airflow/providers/amazon/README.md

```diff
@@ -0,0 +1,344 @@
+| [aws.sensors.s3_prefix.S3PrefixSensor](https://github.com/apache/airflow/blob/master/airflow/providers/amazon/aws/sensors/s3_prefix.py)                                          | [airflow.sensors.s3_prefix_sensor.S3PrefixSensor](https://github.com/apache/airflow/blob/v1-10-stable/airflow/sensors/s3_prefix_sensor.py)                                                  |
+| [aws.sensors.sagemaker_base.SageMakerBaseSensor](https://github.com/apache/airflow/blob/master/airflow/providers/amazon/aws/sensors/sagemaker_base.py)                           | [sensors.sagemaker_base_sensor.SageMakerBaseSensor](https://github.com/apache/airflow/blob/v1-10-stable/airflow/contrib/sensors/sagemaker_base_sensor.py)                                   |
+| [aws.sensors.sagemaker_endpoint.SageMakerEndpointSensor](https://github.com/apache/airflow/blob/master/airflow/providers/amazon/aws/sensors/sagemaker_endpoint.py)               | [sensors.sagemaker_endpoint_sensor.SageMakerEndpointSensor](https://github.com/apache/airflow/blob/v1-10-stable/airflow/contrib/sensors/sagemaker_endpoint_sensor.py)                       |
+| [aws.sensors.sagemaker_transform.SageMakerTransformSensor](https://github.com/apache/airflow/blob/master/airflow/providers/amazon/aws/sensors/sagemaker_transform.py)            | [sensors.sagemaker_transform_sensor.SageMakerTransformSensor](https://github.com/apache/airflow/blob/v1-10-stable/airflow/contrib/sensors/sagemaker_transform_sensor.py)                    |
+| [aws.sensors.sagemaker_tuning.SageMakerTuningSensor](https://github.com/apache/airflow/blob/master/airflow/providers/amazon/aws/sensors/sagemaker_tuning.py)                     | [sensors.sagemaker_tuning_sensor.SageMakerTuningSensor](https://github.com/apache/airflow/blob/v1-10-stable/airflow/contrib/sensors/sagemaker_tuning_sensor.py)                             |
+| [aws.sensors.sqs.SQSSensor](https://github.com/apache/airflow/blob/master/airflow/providers/amazon/aws/sensors/sqs.py)                                                           | [sensors.aws_sqs_sensor.SQSSensor](https://github.com/apache/airflow/blob/v1-10-stable/airflow/contrib/sensors/aws_sqs_sensor.py)                                                           |
+
+
+
+## Hooks
+
+
+### New hooks
+
+| New Airflow 2.0 hooks: `airflow.providers.amazon` package                                                                                              |
+|:-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [aws.hooks.athena.AwsBaseHook](https://github.com/apache/airflow/blob/master/airflow/providers/amazon/aws/hooks/athena.py)                             |
+| [aws.hooks.aws_dynamodb.AwsBaseHook](https://github.com/apache/airflow/blob/master/airflow/providers/amazon/aws/hooks/aws_dynamodb.py)                 |
```

Comment:
Whoops. Probably don't want the base hook apearing multiple times

```

In [7]: airflow.providers.amazon.aws.hooks.aws_dynamodb.AwsBaseHook.__module__                                                                                                                                 
Out[7]: 'airflow.providers.amazon.aws.hooks.base_aws'
```

is how I'd suggest we detect this.

Reply:
Yeah. I missed that one. I manually excluded some of the duplicated ones  - for GCP and few other places where one hook imported another. But it's manual :(. I'd love to automate it.

I could not find an easy way to see it but the __module__ thing might the solution let me try!

Reply:
Yep, The __module__ works, so I can get rid of some hacks :)